easybuild/easyblocks/s/siesta.py

```diff
@@ -0,0 +1,308 @@
+
+    @staticmethod
+    def extra_options(extra_vars=None):
+        """Define extra options for Siesta"""
+        extra = {
+            'with_transiesta': [True, "Build transiesta", CUSTOM],
+            'with_utils': [True, "Build all utils", CUSTOM],
+        }
+        return ConfigureMake.extra_options(extra_vars=extra)
+
+    def configure_step(self):
+        """
+        Custom configure and build procedure for Siesta.
+        - There are two main builds to do, siesta and transiesta
+        - In addition there are multiple support tools to build
+        """
+
+        start_dir = self.cfg['start_dir']
+        obj_dir = os.path.join(start_dir, 'Obj')
+        arch_make = os.path.join(obj_dir, 'arch.make')
+        bindir = os.path.join(self.cfg['start_dir'], 'bin')
+
+        netcdff_loc = get_software_root('NetCDF-Fortran')
+
+        # enable OpenMP support if desired
+        openmp = self.toolchain.options.get('openmp', None)
+        if openmp:
+            scalapack = os.environ['LIBSCALAPACK_MT']
+            blacs = os.environ['LIBSCALAPACK_MT']
+            lapack = os.environ['LIBLAPACK_MT']
+            blas = os.environ['LIBBLAS_MT']
+        else:
+            scalapack = os.environ['LIBSCALAPACK']
+            blacs = os.environ['LIBSCALAPACK']
+            lapack = os.environ['LIBLAPACK']
+            blas = os.environ['LIBBLAS']
+
+        regex_subs = [
+            ('dc_lapack.a', ''),
+            (r'^NETCDF_INTERFACE\s*=.*$', ''),
+            ('libsiestaBLAS.a', ''),
+            ('libsiestaLAPACK.a', ''),
+        ]
+
+        # Make a temp installdir during the build of the various parts
+        try:
+            mkdir(bindir)
+        except OSError, err:
+            raise EasyBuildError("Failed to create temp installdir %s: %s", bindir, err)
+
+        # change to actual build dir
+        try:
+            os.chdir(obj_dir)
+        except OSError, err:
+            raise EasyBuildError("Failed to change to build dir: %s", err)
+
+        # Populate start_dir with makefiles
+        run_cmd(os.path.join(start_dir, 'Src', 'obj_setup.sh'), log_all=True, simple=True, log_output=True)
+
+        if LooseVersion(self.version) < LooseVersion("4.1-b2"):
+            # MPI?
+            if self.toolchain.options.get('usempi', None):
+                self.cfg.update('configopts', '--enable-mpi')
+
+            # BLAS and LAPACK
+            self.cfg.update('configopts', '--with-blas="%s"' % blas)
+            self.cfg.update('configopts', '--with-lapack="%s"' % lapack)
+
+            # ScaLAPACK (and BLACS)
+            self.cfg.update('configopts', '--with-scalapack="%s"' % scalapack)
+            self.cfg.update('configopts', '--with-blacs="$%s"' % blacs)
+
+            # NetCDF-Fortran
+            if netcdff_loc:
+                self.cfg.update('configopts', '--with-netcdf=-lnetcdff')
+
+            super(EB_Siesta, self).configure_step(cmd_prefix='../Src/')
+
```

Comment:
that's weird...

maybe include a comment on that above the lines below to make that clear?